onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -1206,6 +1206,33 @@ ORT_API_STATUS_IMPL(OrtApis::GetStringTensorElementLength, _In_ const OrtValue*
   API_IMPL_END
 }
 
+ORT_API_STATUS_IMPL(OrtApis::GetTensorSizeInBytes, _In_ const OrtValue* value, _Out_ size_t* out) {
+  API_IMPL_BEGIN
+
+  if (value == nullptr) {
+    return OrtApis::CreateStatus(ORT_INVALID_ARGUMENT, "Input `value` argument must not be null");
+  }
+
+  if (out == nullptr) {
+    return OrtApis::CreateStatus(ORT_INVALID_ARGUMENT, "Output `size` argument must not be null");
```

Comment:
nit: if we're calling it `size` in the message, it seems odd that the actual variable is named `out`. can they be consistent?

Reply:
Q: Is it common practice in ORT to check for invalid out parameters? I know in COM API's, they often (intentionally to catch contract violations early) write up-front.

e.g. 
```c++
HRESULT SomeComApi(_Out_ Foo* foo, _Out_t UINT32* size)
{
    *foo = nullptr;
    *size = 0;

    ...
}
```

Reply:
No, it is not common. I have not seen a policy on this.